### PR TITLE
Prep for 2.0.2 release

### PR DESCRIPTION
**Pull request by bryevdv**
This PR streamlines setup.py:

* relegate package data specification to MANIFEST.in
* update to versioneer 0.18
* remove ancient get_cmdclass cruft
* only scans for `bokeh` packages

Will also add 2.0.2 release notes to finish prep for release. 

Leaving `release` top level dir for now, but the new config excludes every pacakge that is not `bokeh` or `bokeh.*`

### CODE REVIEW

File: bokeh/_version.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def get_config():
     # _version.py
     cfg = VersioneerConfig()
     cfg.VCS = "git"
-    cfg.style = "git-describe"
+    cfg.style = "pep440"
```
Comment (bryevdv) [inline bokeh/_version.py:43]: @mattpap our `setup.cfg` has "pep440" but this cfg is only consulted when versioneer is *installed* so evidently it was never actually used (until now, if we keep the change on the line above). Do you have an opinion? The difference between the two is outlined here:

https://github.com/warner/python-versioneer/blob/master/details.md#how-do-i-select-a-version-style

File: bokeh/_version.py [hunk 2]
```diff
@@ -234,15 +234,10 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
...
-    # argument since tag_prefix is empty in the bokeh setup config.
-    describe_args = ["describe", "--tags", "--dirty", "--always", "--long"]
-
-    if tag_prefix:
-        describe_args.extend(["--match", "%s*" % tag_prefix])
-    describe_out, rc = run_command(GITS, describe_args, cwd=root)
+    describe_out, rc = run_command(GITS, ["describe", "--tags", "--dirty",
+                                          "--always", "--long",
+                                          "--match", "%s*" % tag_prefix],
+                                   cwd=root)
```
Comment (bryevdv) [inline bokeh/_version.py:240]: I will be testing an sdist on windows to see if this is still any kind of issue

File: bokeh/_version.py (hunk 1 above)
Comment (mattpap) [reply bokeh/_version.py:43]: I recall both styles being mixed in some situations in the past (e.g. bokehjs has `[+-]` in a regex to accommodate this). I don't have preference as long as things are consistent, though `git-describe` seems cleaner to me.

Comment (bryevdv) [reply bokeh/_version.py:43]: OK I am happy to leave it `git-describe` I just want to make sure the `setup.cfg` matches our intentions. I will revert this and update the value the config.

File: bokeh/_version.py (hunk 2 above)
Comment (bryevdv) [reply bokeh/_version.py:240]: I built/installed/smoke-tested an sdist on windows and everything went fine

**Pull request merged**